plugins/landing-page/gatsby-node.ts

```diff
@@ -12,6 +12,8 @@ import '@dvcorg/gatsby-theme-iterative/config/prismjs/dvc'
 
 import yaml from 'js-yaml'
 
+const unusedVariable = "Oh no!"
+
```

Comment:
unused variables aren't automatically fixed, so the workflow will simply fail until the user fixes them.